lib/livebook/storage/ets.ex

```diff
@@ -72,10 +76,18 @@ defmodule Livebook.Storage.Ets do
   end
 
   @impl GenServer
-  def init(_opts) do
-    table = :ets.new(@table_name, [:named_table, :protected, :duplicate_bag])
+  def init(opts) do
+    # enable passing table name for testing purposes
+    table_name = Keyword.get(opts, :table_name, @table_name)
+    table = :ets.new(table_name, [:named_table, :protected, :duplicate_bag])
+
+    # NOTE: should we always start persisting to file or make user state that explicitly?
+    secret = Keyword.get(opts, :secret, default_secret())
```

Comment:
@josevalim this goes back to our recent discussions, at this point I'd say we can have application env (set via env var) for the secret, we read the application env and encrypt only if it's there. I'm saying application env, because we may want to configure it at runtime at some point. wdyt?